Unity-C#/README.md

```diff
@@ -0,0 +1,263 @@
+#Preamble 
+The below style uses the MSDN Style guidelines as a base to work with Unity, however because some of the nuances in Unity this style adapts and breaks from some of the standard conventions to obtain a higher level of readability.
```

Comment:
From my reading this document can stand on its own. I feel the MSDN reference should come at the document's bottom and be treated as "Anything not defined here should defer to MSDN".

On the whole I think the frequent references to MSDN and justifications for departures from MSDN style should be in a separate section along with the references.

I think the document would also benefit from a section targeted at JS programmings. This section would explain what is different and call out features of C# which require special attention.

Thus in concrete terms I think this document should not depend on MSDN. Most sections should explain the WHAT and the WHY but drop the WHY NOT MSDN. The WHY NOT MSDN could be useful in a dedicated section.